surprise/__init__.py

```diff
@@ -1,5 +1,3 @@
-from pkg_resources import get_distribution
-
```

Comment:
```suggestion

Reply:
See discussion here in this PR: https://github.com/NicolasHug/Surprise/pull/476#discussion_r1595421963